src/syrupy/report.py

```diff
@@ -66,6 +82,10 @@ def __attrs_post_init__(self) -> None:
                 else:
                     self.failed.update(snapshot_fossil)
 
+    @property
+    def ran_test_files(self) -> Optional[List[str]]:
```

Comment:
This is not optional since it'll alway return a list no?
```suggestion
    def ran_test_files(self) ->List[str]:
```

Reply:
Indeed.